Formula/jadx.rb

```diff
@@ -0,0 +1,38 @@
+class Jadx < Formula
+  desc "Dex to Java decompiler"
+  homepage "https://github.com/skylot/jadx"
+  url "https://github.com/skylot/jadx/releases/download/v0.6.0/jadx-0.6.0.zip"
+  sha256 "0baa680c8bb4f1c49b6d8b94b3947b8fedf263b5a0c20d3b38da35b57b857228"
+
+  head do
+    url "https://github.com/skylot/jadx.git"
+    depends_on :java => "1.8"
```

Comment:
This has been addressed in https://github.com/Homebrew/homebrew-core/pull/1515/commits/de6e16240ccfca0d412cb4513e5dd768e3951852 - thank you for noticing that.